deals/deals_test.go

```diff
@@ -25,7 +25,7 @@ func TestMain(m *testing.M) {
 }
 
 func TestStore(t *testing.T) {
-	numMiners := []int{1, 3, 5}
+	numMiners := []int{1, 3}
```

Comment:
Reducing heaviness despite in my laptop this never seemed to be a problem.
I'll take a look if there's something extra to do CI-wise... I don't want to set-up a self-hosted costly thing for running this thing, seems like too much. On the other hand, don't like the idea of relying on us to make heavier tests...